src/utils/chain.ts

```diff
@@ -0,0 +1,10 @@
+export const EIPNumberPrefix = 'eip155:'
+
+/**
```

Comment:
I think we should be using JSDoc for the utility classes. It's helpful to understand it especially for the complex ones